third_party/openenclave/Makefile

```diff
@@ -96,6 +96,8 @@ endif
 	mkdir -p $(TOP)/build/bin
 	rm -f $(TOP)/build/bin/myst-gdb
 	ln -s $(TOP)/build/openenclave/bin/oegdb $(TOP)/build/bin/myst-gdb
+	# Patch minimum required lldb version from 7 to 8.
```

Comment:
Should we update this in oelldb's source in OE's mystikos branch instead?

Reply:
That can be done separately as noted in the comment. I'm trying to avoid unnecessarily modifying the OE branch.